src/Domain/Event/AlbumCreatedEvent.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Event;
+
+use App\Entity\Album;
+use Sulu\Bundle\ActivityBundle\Domain\Event\DomainEvent;
+
+class AlbumCreatedEvent extends DomainEvent
+{
+    private Album $album;
+
+    /**
+     * @var mixed[]
+     */
+    private array $payload;
+
+    /**
+     * @param mixed[] $payload
+     */
+    public function __construct(Album $album, array $payload)
+    {
+        parent::__construct();
+
+        $this->album = $album;
+        $this->payload = $payload;
+    }
+
+    public function getAlbum(): Album
+    {
+        return $this->album;
+    }
+
+    public function getEventPayload(): ?array
+    {
+        return $this->payload;
```

Comment:
By configuring `sulu_activity.storage.persist_payload` to `true` in `config/packages/sulu_activity.yaml`, the event payload will be persisted in the database too.